cmd/medius/images/push.go

```diff
@@ -261,8 +303,22 @@ func (b *buildAndPublish) pushImage(containerDisk v1.Image, name string) error {
 	return nil
 }
 
+func (b *buildAndPublish) pushImageIndex(containerDiskIndex v1.ImageIndex, name string) error {
+	if !b.Options.DryRun {
```

Comment:
Use early return instead of if/else.

Reply:
It's similar to the function above, `pushImage`. Do you want to change it in both places?